### ISSUE TITLE
Multilingual root routes

### ISSUE THREAD

**Issue by ionlizarazu** [plone/volto#2577]
**Some routes should be language rooted instead of site-rooted on multilingual sites.**
Some routes with public views as /sitemap should be language rooted. I think we need for example /en/sitemap and /es/sitemap on multilingual sites instead of just /sitemap

**multilingualRoutes on routes.js**
I propose to have another variable called multilingualRoutes as same as defaultRoutes, but adding it only when config.settings.isMultilingual is true.

**multilingualRoutes example for sitemap**
```
export const multilingualRoutes = [{
      path: `/(${config.settings.supportedLanguages.join('|')})/sitemap`,
      component: Sitemap,
    }]
```
**add multilingualRoutes to routes**
```
routes: [
...
    ...(config.addonRoutes || []),
    ...defaultRoutes,
    ...(config.settings.isMultilingual && multiligualRoutes),
]
```

**Comment by tiberiuichim** [comment]
Your proposal looks good to me. I think it should be:

```
routes: [
...
    ...(config.addonRoutes || []),
    ...((config.settings.isMultilingual && multiligualRoutes) || []),
    ...defaultRoutes,
]
```

**Comment by ionlizarazu** [comment]
Solved with PR #2578
